core/src/main/java/org/elasticsearch/search/SearchModule.java

```diff
@@ -477,14 +483,9 @@ public PipelineAggregationSpec addResultReader(String writeableName, Writeable.R
     @Override
     protected void configure() {
         if (false == transportClient) {
-            /*
-             * Nothing is bound for transport client *but* SearchModule is still responsible for settings up the things like the
-             * NamedWriteableRegistry.
-             */
             bind(IndicesQueriesRegistry.class).toInstance(queryParserRegistry);
```

Comment:
Now that you are binding SearchRequestParsers do we need to bing IndicesQueriesRegistry?